### DISCUSSION TITLE
Book Lib Connect Exception

### DISCUSSION THREAD

**Discussion by Sailfish** [audiamus/AaxAudioConverter#107] [General]
After launching Book Lib Connect I get an Unhandled Exception (see attached BookLibException.jpg). This exception occurs after the program window displays even if no further action is done on my part, see attached BookLinConnectStartPage.jpg). It doesn't immediately fire but takes about 30 seconds to occur.

The exception dialog gives the option to Continue or Quit; however, if I choose Continue, the program goes into what seems like an infinite loop (quit via Task Manager after waiting 2 hours.)

I've also attached several program settings files (usersettings.json, appsettings.json were appended with .txt to allow attachment), The full text of the exception dialog, as well as the -log=3 log file.

Let me know I you need anything else.

Regards,
Sailfish

[appsettings.json.txt](https://github.com/audiamus/AaxAudioConverter/files/7886555/appsettings.json.txt)
[usersettings.json.txt](https://github.com/audiamus/AaxAudioConverter/files/7886562/usersettings.json.txt)
[BookLibConnect_2022-01-17_001.log](https://github.com/audiamus/AaxAudioConverter/files/7886565/BookLibConnect_2022-01-17_001.log)
[ExceptionFULL.txt](https://github.com/audiamus/AaxAudioConverter/files/7886572/ExceptionFULL.txt)
) as well as the -log=3 log file.
![BookLibException](https://user-images.githubusercontent.com/110050/149879329-21be7685-61b0-4e17-b7c1-9ffdb3743d1f.jpg)
![BookLinConnectStartPage](https://user-images.githubusercontent.com/110050/149879369-ec22ac6e-0d10-4275-87d9-97d026d539f2.jpg)

**Comment by audiamus** [comment]
Thanks. It appears to be the same problem as this one: https://github.com/audiamus/BookLibConnect/issues/1

Most probably a non-numeric sequence ID of a title in a book series. I'll fix it in the coming days.

_PS: Never upload the configuration file. Fortunately, it's encrypted by default._
